pkg/controller/promotionreconciler/reconciler.go

```diff
@@ -32,47 +32,36 @@ import (
 )
 
 type Options struct {
-	DryRun                bool
 	CIOperatorConfigAgent agents.ConfigAgent
-	ConfigGetter          config.Getter
 	GitHubClient          github.Client
-	// The registryManager is set up to talk to the cluster
-	// that contains our imageRegistry. This cluster is
-	// most likely not the one the normal manager talks to.
-	RegistryManager controllerruntime.Manager
 }
 
 const ControllerName = "promotionreconciler"
 
-func AddToManager(mgr controllerruntime.Manager, opts Options) error {
+func AddToManager(mgr controllerruntime.Manager, opts Options, enqueuer prowjobreconciler.Enqueuer) error {
 	// Pre-Allocate the Image informer rather than letting it allocate on demand, because
 	// starting the watch takes very long (~2 minutes) and having that delay added to our
 	// first (# worker) reconciles skews the workqueue duration metric bigtimes.
-	if _, err := opts.RegistryManager.GetCache().GetInformer(context.TODO(), &imagev1.Image{}); err != nil {
+	if _, err := mgr.GetCache().GetInformer(context.TODO(), &imagev1.Image{}); err != nil {
```

Comment:
This is wrong I think, this needs to be the api.ci manager and its the app.ci one now
This part can be just deleted though, it happens in `imagestreamwarpper.MustNew`

Reply:
removed